docs/developer-docs/latest/setup-deployment-guides/configurations/required/databases.md

```diff
@@ -275,7 +280,249 @@ await pluginStore.set({
 });
 ```
 
-## Databases installation guides
+## Environment variables in database configurations
+
+Strapi version v4.6.2 and higher includes the database configuration options in the `./config/database.js` or `./config/datbase.ts` file. When a new project is created the environment variable `DATABASE_CLIENT` with the value `mysql`, `postgres`, or `sqlite` is automatically added to the `.env` file depending on which database you choose during project creation. Additionally, all of the environment variables necessary to connect to your local development database are also added to the `.env` file.  The following is an example of the generated config file:
```

Comment:
```suggestion
Strapi version v4.6.2 and higher includes the database configuration options in the `./config/database.js` or `./config/datbase.ts` file. When a new project is created the environment variable `DATABASE_CLIENT` with the value `mysql`, `postgres`, or `sqlite` is automatically added to the `.env` file depending on which database you choose during project creation. Additionally, all of the environment variables necessary to connect to your local development database are also added to the `.env` file.  The following is an example of the generated configuration file:
```
FYI, Grammarly also suggests the following improvements 👀 :
![Screenshot 2023-02-15 at 14 05 41](https://user-images.githubusercontent.com/4233866/219035508-5bfb2a6b-6215-4628-a30d-c14f689aef7d.png)